tests/test_visualizations.py

```diff
@@ -193,3 +193,18 @@ def test_playrates():
             filename, 1)
     os.remove(filename)
     plt.close('all')
+
+
+def test_to_html():
+    """Test converting an animation to HTML."""
+    try:
+        from IPython.display import HTML
+    except ImportError:
+        pytest.skip('Cannot convert movie to HTML without IPython.')
+
+    nx, ny, nt = 10, 10, 50
+    sta = utils.create_spatiotemporal_filter(nx, ny, nt)[-1]
+    anim = viz.play_sta(sta)
+    html = viz.to_html(anim)
+    assert isinstance(html, HTML)
+
```

Comment:
Will do. I'll also add a `v0.5` file to changelogs. If you're on board, we can bump the version to 0.5 when merging this?